### DISCUSSION TITLE
Does the anonymizer script support all functions that exist in CTP?

### DISCUSSION THREAD

**Discussion by kirbyju** [RSNA/anonymizer#8] [General]
Are there any functions described on https://mircwiki.rsna.org/index.php?title=The_CTP_DICOM_Anonymizer that do not work in Anonymizer?

**Comment by mdevans** [comment]
Yes, this version of the Anonymizer implements a subset of the functions in
the CTP Java Anonymizer, please refer to the [Overview/Differences from Java Version] here:
https://rsna.github.io/anonymizer/

On Thu, 1 Aug 2024 at 19:12, Justin Kirby ***@***.***> wrote:

> Are there any functions described on
> https://mircwiki.rsna.org/index.php?title=The_CTP_DICOM_Anonymizer that
> do not work in Anonymizer?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/RSNA/anonymizer/issues/3>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIXFIGUYRVBPHAEB2RB2FDZPJUAPAVCNFSM6AAAAABL27LZHCVHI2DSMVQWIX3LMV43ASLTON2WKOZSGQ2DEOJWGQ3TGMA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Michael Evans
*DX.LIFE (Pty) Ltd.*
+27 72 584 6989
